draft-ietf-tls-extended-key-update.md

```diff
@@ -512,13 +512,102 @@ Auth | {CertificateVerify}
 
 #  DTLS 1.3 Considerations
 
+Unlike TLS 1.3, DTLS 1.3 implementations must take into account that handshake
+messages are not transitted over a reliable transport protocol. As with other
+handshake messages with no built-in response, NewKeyUpdate messages MUST be
+acknowledged. Unlike the NewKeyUpdate message, the ExtendedKeyUpdateRequest
+is acknowledged by the ExtendedKeyUpdateResponse.
+
+The exchange has the following steps:
+
+1. Initiator sends a ExtendedKeyUpdateRequest message, which contains
+a key share. While an extended key update is in progress, the initiator
+MUST NOT initiate further key updates.
+
+2. On receipt of the ExtendedKeyUpdateRequest message, the responder
+sends the ExtendedKeyUpdateResponse message. If the responder accepts the
+request, it sets the status to `accepted` and includes its own key share.
+If the responder declines the request, it sets the status accordingly and
+does not include the key share. While an extended key update is in progress,
+the responder MUST NOT initiate further key updates.
+
+3. On receipt of the ExtendedKeyUpdateResponse message with `accepted` status,
+the initiator is able to derive a secret key based on the exchanged key shares.
+The NewKeyUpdate message is intentionally an empty structure that triggers
+the transition to new keying material.
+
+4. The initiator transmits the NewKeyUpdate message.
+
+5. On receipt of the NewKeyUpdate message, the responder MUST update its receive keys.
+In response, the responder acknowledges the received NewKeyUpdate
+message and transmits a NewKeyUpdate message. That message will acknowledge
```

Comment:
I can't parse the last line.

Reply:
Tried to fix the text. Hope it is understandable now...